deps/exokit-bindings/canvascontext/src/canvas-context.cc

```diff
@@ -361,6 +391,8 @@ NAN_METHOD(CanvasRenderingContext2D::New) {
   Nan::SetAccessor(ctxObj, JS_STR("textBaseline"), TextBaselineGetter, TextBaselineSetter);
   Nan::SetAccessor(ctxObj, JS_STR("direction"), DirectionGetter, DirectionSetter);
 
+  ctxObj->Set(JS_STR("font"), JS_STR("10px sans-serif"));
```

Comment:
This is Chrome's default font for freshly-created canvases.

Reply:
If it works!